src/string/get_longest_palindrome.rs

```diff
@@ -0,0 +1,50 @@
+/*
+Find the longest palindrome substring in the string
+dp[i][j] show s[i]..s[j] a palindrome
+so dp[i][j] = (s[i]==s[j]) && dp[i+1][j-1]
+*/
+
+pub fn get_longest_palindrome(s: &str) -> String {
+    if s.is_empty() {
+        return "".to_string();
+    }
+    let n = s.chars().count();
+    let s: Vec<char> = s.chars().collect();
```

Comment:
You can avoid one iteration across `s`:
```suggestion
    let s: Vec<char> = s.chars().collect();
    let n = s.len();
```